### ISSUE TITLE
Probabilities in `preds` must sum up to 1 across the `C` dimension.

### ISSUE THREAD

**Issue by huhubrave** [PyTorchLightning/metrics#264]
## 🐛 Bug
At **16 accuracy**, When use softmax, Probabilities in `preds`  **can not** sum up to 1 across the `C` dimension.
You may not need to check like this.  in [_check_classification_inputs](https://github.com/PyTorchLightning/metrics/blob/master/torchmetrics/utilities/checks.py)
`if not torch.isclose( preds.sum(dim=1), torch.ones_like(preds.sum(dim=1))).all():    
            raise ValueError("Probabilities in `preds` must sum up to 1 across the `C` dimension.")`

###   To Reproduce
from torchmetrics import Accuracy
import torch
target = torch.tensor([0, ] ,device ='cuda:0')
preds = torch.tensor([[0.1962, 0.1890, 0.2385, 0.1919, 0.10]] ,dtype=torch.float16 ,device ='cuda:0')
preds =  torch.softmax( preds , axis=-1)
accuracy = Accuracy().cuda()
accuracy(preds, target)

**Comment by SkafteNicki** [comment]
Hi @huhubrave,
The check was removed in this PR https://github.com/PyTorchLightning/metrics/pull/200 to support unnormalized scores. 
Please upgrade to master: pip install git+https://github.com/PytorchLightning/metrics.git@master and it should work for you :]

**Comment by huhubrave** [comment]
Thank you for your reply. It works very well.
